src/Spice86/ViewModels/DebugWindowViewModel.cs

```diff
@@ -61,11 +66,34 @@ public partial class DebugWindowViewModel : ViewModelBase, IInternalDebugger {
         CpuViewModel = new(pauseStatus);
         MidiViewModel = new();
         MemoryViewModel = new(pauseStatus);
-        Dispatcher.UIThread.Post(() => rootComponent.Accept(this), DispatcherPriority.Background);
+        Dispatcher.UIThread.Post(() => programExecutor.Accept(this), DispatcherPriority.Background);
+    }
+    
+    [RelayCommand]
+    public void Pause() {
+        if (_programExecutor is null || _pauseStatus is null) {
+            return;
+        }
+        _pauseStatus.IsPaused = _programExecutor.IsPaused = IsPaused = true;
+    }
+
+    [RelayCommand(CanExecute = nameof(IsPaused))]
```

Comment:
`IsPaused ` does not exist in this class, so the Continue button does not work anymore.

Reply:
The IsPaused property is source generated by the ObservableProperty attribute.

What was missing on the _isPaused field, was:

    [NotifyCanExecuteChangedFor(nameof(ContinueCommand))]